kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -756,3 +756,10 @@ pwdStrengthNonAlphanumericRequired="Password must contain at least one non alpha
 pwdStrengthBothCasesRequired="Password must contain both uppercase and lowercase characters"
 
 logDownload=Click the following button to download all system logs in a single zip file
+
+configurationChangeEvent_Created=created {0}
+configurationChangeEvent_Removed=removed {0}
+configurationChangeEvent_Updated=updated {0}
+configurationChangeEvent_Generic=modified some configurations
+configurationChangeEvent_Rollback=performed a rollback
+configurationChangeEventNotification=Another user has {0}. You might see an inconsistent state. Please refresh the page.
```

Comment:
Maybe it is better to say "A user has {0}..."?

> You might see an inconsistent state. Please refresh the page.

"You might see an inconsistent state and a page refresh might be needed to show the latest changes."
Better in this way?